packages/nimble-components/src/rich-text/editor/index.ts

```diff
@@ -343,6 +343,7 @@ export class RichTextEditor extends FoundationElement implements ErrorPattern {
          */
         return new Editor({
             element: this.editor,
+            enableInputRules: [BulletList, OrderedList],
```

Comment:
I think a comment will be helpful here
```suggestion
            // The editor will detect markdown syntax for only these items
            // https://tiptap.dev/api/editor#enable-input-rules
            enableInputRules: [BulletList, OrderedList],
```

Reply:
Added